src/include/btree_inline.h

```diff
@@ -1756,6 +1757,16 @@ __wt_page_release(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t flags)
     if (F_ISSET(btree, WT_BTREE_IN_MEMORY))
         return (0);
 
+    /*
+     * If it is switched on, the session debug "release evict page configuration" will evict pages
```

Comment:
The comment may be missing something. How about `Attempt to evict the page if the session is configured with release evict page debug option`?